meson.build

```diff
@@ -982,6 +982,9 @@ if default_keymap == ''
 endif
 conf.set_quoted('SYSTEMD_DEFAULT_KEYMAP', default_keymap)
 
+default_timezone = get_option('firstboot-timezone')
+conf.set_quoted('SYSTEMD_FIRSTBOOT_TIMEZONE', default_timezone)
```

Comment:
I do not think we should provide such compiled default value. That's mostly useless, except for when e.g. systemd is built for a local company or so.

If you want to introduce a way to provide a default timezone. Please consider to use an environment variable or a kernel command line option.

Reply:
Yea I wasn't sure about this. I kinda did it so the default can be changed, but it was more of a stopgap solution since the suggested plan was to have the default fallback (where this is used) be determined by the entered locale or something. So I just removed it.